requre/polarion/__init__.py

```diff
@@ -27,7 +26,14 @@
 sessions.Session.send = RequestResponseHandling.decorator(
     item_list=[1])(sessions.Session.send)
 
-tmt.utils.check_git_url = Guess.decorator_plain()(tmt.utils.check_git_url)
+# The function moved to 'tmt.utils.git'
+# https://github.com/teemtee/tmt/pull/2983
+try:
+    import tmt.utils
+    tmt.utils.check_git_url = Guess.decorator_plain()(tmt.utils.check_git_url)
+except AttributeError:
+    import tmt.utils.git
+    tmt.utils.git.check_git_url = Guess.decorator_plain()(tmt.utils.git.check_git_url)
```

Comment:
Could the failing tests be caused by the function being moved? Maybe this could be the correct setup:

```python
import tmt.utils
import tmt.utils.git
tmt.utils.check_git_url = Guess.decorator_plain()(tmt.utils.git.check_git_url)
```

Reply:
Tests are failing because of `tmt-1.35` has not reached stable repos. Passing `rawhide` should be enough.